documentation/coding_guidelines.md

```diff
@@ -227,6 +227,16 @@ impl Foo {
 }
 ```
 
+### Integer Arithmetic
+
+As every integer operation (`+`, `-`, `/`, `*`, etc.) implies edge-cases (e.g. `u64::MAX + 1`),
```

Comment:
divide can result in edge cases in integers :O?

Reply:
dividing by 0